pkg/agent/option.go

```diff
@@ -56,6 +56,13 @@ type ClusterRegistrationOptions struct {
 	// No tunnel logging by default
 	TunnelLogging bool
 
+	// Estimator set estimator mode, eg. disable, default, inChildCluster, inParentCluster
```

Comment:
If the estimator address is empty, it mean estimator is not available.

The estimator is running either agent-side or hub-side. So we only need to add a boolean flag to indicate this.